python/pyspark/worker.py

```diff
@@ -176,6 +180,16 @@ def wrap_arrow_batch_udf(f, args_offsets, kwargs_offsets, return_type, runner_co
     elif type(return_type) == BinaryType:
         result_func = lambda r: bytes(r) if r is not None else r  # noqa: E731
 
+    if zero_arg_exec:
+
+        def get_args(*args: pd.Series):
+            return [() for _ in range(len(args[0]))]
```

Comment:
Can we directly use `args[0]`?

```suggestion
            return [() for _ in range(len(args[0]))]
            return [() for _ in args[0]]
```